### ISSUE TITLE
Log: "Disabling Kelvin" "Activating Kelvin" loop, and overriding custom scenes

### ISSUE THREAD

**Issue by Katjubu** [stefanwichmann/kelvin#37]
I'm unable to select custom scenes or change anything during the daytime without Kelvin throwing a log loop and overriding what I'm trying to do. I reset the hub (clean up in Hue app) with no change. Kelvin is running as a service on Ubuntu. I would like it to automatically enable when the lights are on after sunset but be able to control the lights myself through the day. Thanks for your help!

Log:
```
Jun 15 07:56:21 kelvin kelvin[2072]: time="2018/15/06 07:56:21" level=info msg="💡 Light Skull - Light state has been changed manually. Disabling Kelvin..."
Jun 15 07:56:24 kelvin kelvin[2072]: time="2018/15/06 07:56:24" level=info msg="💡 Light Side - Detected matching target state. Activating Kelvin..."
```
Debug:
```
DEBU[2018/15/06 07:52:36] 💡 Light Side - Initialization complete. Identified as Color temperature light (ModelID: LTW004, Version: 5.105.0.21536)
INFO[2018/15/06 07:52:36] 💡 Light Side - Activating schedule for Jun 15 2018 (Sunrise: 05:27, Sunset: 20:32)
INFO[2018/15/06 07:52:36] 💡 Light Side - Activating interval 05:27 - 20:32
DEBU[2018/15/06 07:52:36] 💡 Light Side - The calculated lightstate for the interval 05:27 - 20:32 is {ColorTemperature:-1 Brightness:-1}
DEBU[2018/15/06 07:52:36] 💡 Light Side - Initialized target light state to {ColorTemperature:-1 Brightness:-1}
DEBU[2018/15/06 07:52:36] 💡 Light Side - Starting cyclic update...
INFO[2018/15/06 07:52:37] 💡 Light Side - Light just appeared.
INFO[2018/15/06 07:52:37] 💡 Light Side - Initializing state to -1K at -1% brightness.
DEBU[2018/15/06 07:52:37] 💡 Light Side - Adjusted color temperature to light capability of 2000K
DEBU[2018/15/06 07:52:37] 💡 HueLight Side - Setting light state to 2000K and -1% brightness.
DEBU[2018/15/06 07:52:38] 💡 HueLight Side - Color temperature has changed! Current light state: &{Name:Side HueLight:{Id:5 Name:Side bridge:0xb26b80} SetColorTemperature:2000 SetBrightness:-1 TargetColorTemperature:500 TargetColor:[0.527 0.404] TargetBrightness:-1 CurrentColorTemperature:454 CurrentColor:[] CurrentBrightness:254 CurrentColorMode:ct SupportsColorTemperature:true SupportsXYColor:false Dimmable:true Reachable:true On:true MinimumColorTemperature:2000}
WARN[2018/15/06 07:52:38] 💡 HueLight Side - Failed to update light state: &{Name:Side HueLight:{Id:5 Name:Side bridge:0xb26b80} SetColorTemperature:2000 SetBrightness:-1 TargetColorTemperature:500 TargetColor:[0.527 0.404] TargetBrightness:-1 CurrentColorTemperature:454 CurrentColor:[] CurrentBrightness:254 CurrentColorMode:ct SupportsColorTemperature:true SupportsXYColor:false Dimmable:true Reachable:true On:true MinimumColorTemperature:2000}
DEBU[2018/15/06 07:52:38] 💡 Light Side - Adjusted color temperature to light capability of 2000K
```
Config:
```
{
  "version": 1,
  "bridge": {
    "ip": "192.168.1.52",
    "username": ""
  },
  "location": {
    "latitude": ,
    "longitude": 
  },
  "webinterface": {
    "enabled": false,
    "port": 8080
  },
  "schedules": [
    {
      "name": "default",
      "associatedDeviceIDs": [
        2,
        3,
        5,
        6
      ],
      "enableWhenLightsAppear": true,
      "defaultColorTemperature": -1,
      "defaultBrightness": -1,
      "beforeSunrise": [
        {
          "time": "4:00",
          "colorTemperature": 2200,
          "brightness": 1
        }
      ],
      "afterSunset": [
        {
          "time": "20:00",
          "colorTemperature": 3400,
          "brightness": 50
        },
        {
          "time": "22:00",
          "colorTemperature": 2200,
          "brightness": 1
        }
      ]
    }
  ]
}
```

**Comment by stefanwichmann** [comment]
Thanks for reporting this! I think this might be the same issue as #39.

Working on it!

**Comment by stefanwichmann** [comment]
Please try again with release 1.1.8. Thanks!

**Comment by Katjubu** [comment]
Unfortunately it seems to work the same. Setting color or brightness to -1 will keep enabling and disabling Kelvin. Setting a prefered brightness and color (100% and 5500k) works but then I am unable to use scenes without Kelvin trying to change them.

**Comment by stefanwichmann** [comment]
Ok, there is another issue here. I am working on a fix and will try to push another release tonight. Thanks for testing and reporting! This helps a lot!

**Comment by stefanwichmann** [comment]
I just released v1.1.9 containing two fixes for scene detection. Please give it a try.
